iothub_client/tests/iothubdeviceclient_ll_ut/iothubdeviceclient_ll_ut.c

```diff
@@ -410,5 +411,18 @@ TEST_FUNCTION(IoTHubDeviceClient_LL_UploadMultipleBlocksToBlob_Test)
 
 #endif // !DONT_USE_UPLOADTOBLOB
 
+TEST_FUNCTION(IoTHubDeviceClient_LL_SendMessageDisposition_Test)
+{
+    //arrange
+    umock_c_reset_all_calls();
+    STRICT_EXPECTED_CALL(IoTHubClientCore_LL_SendMessageDisposition(IGNORED_PTR_ARG, TEST_MESSAGE_HANDLE, IOTHUBMESSAGE_ACCEPTED));
+
+    //act
+    IOTHUB_CLIENT_RESULT result = IoTHubDeviceClient_LL_SendMessageDisposition(TEST_IOTHUB_DEVICE_CLIENT_LL_HANDLE, TEST_MESSAGE_HANDLE, IOTHUBMESSAGE_ACCEPTED);
+
+    //assert
+    ASSERT_ARE_EQUAL(char_ptr, umock_c_get_expected_calls(), umock_c_get_actual_calls());
+    ASSERT_IS_TRUE(IOTHUB_CLIENT_OK == result);
+}
 
 END_TEST_SUITE(iothubdeviceclient_ll_ut)
```

Comment:
nit: newline even though not part of this PR

Reply:
Fixed.